src/warnet/server.py

```diff
@@ -269,7 +278,8 @@ def network_export(self, network: str) -> str:
         Export all data for sim-ln to subdirectory
         """
         try:
-            wn = Warnet.from_network(network)
+            wn = self.get_warnet(network)
+            self.warnets[network] = wn
```

Comment:
why would you need to set this `wn` again?

Reply:
That is a good question, I think it's a mistake.